src/views/secondary-bar/secondary-bar.tsx

```diff
@@ -240,13 +227,7 @@ export const ShellSecondaryBar: React.VFC<ShellSecondaryBarProps> = ({ expanded
 				[]
 			);
 
-			const mixedRootsAndFilters = orderBy(
-				[...rootItems, ...filtersAsRoots],
-				['priority'],
-				['asc']
-			);
-
-			return [...mixedRootsAndFilters, filters, uploads];
+			return orderBy([...rootItems, ...filters, uploads], ['priority'], ['asc']);
 		}
 		return [
```

Comment:
We missed the sort of the fallback return. I think we could refactor this code too in order to use the same orderBy function of the previous return.